src/main/java/slimeknights/tconstruct/gadgets/entity/shuriken/FlintShurikenEntity.java

```diff
@@ -0,0 +1,37 @@
+package slimeknights.tconstruct.gadgets.entity.shuriken;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.Item;
+import net.minecraft.world.World;
+import slimeknights.tconstruct.gadgets.TinkerGadgets;
+
+public class FlintShurikenEntity extends ShurikenEntityBase {
+
+  public FlintShurikenEntity(EntityType<? extends FlintShurikenEntity> p_i50159_1_, World p_i50159_2_) {
+    super(p_i50159_1_, p_i50159_2_);
+  }
+
+  public FlintShurikenEntity(World worldIn, LivingEntity throwerIn) {
+    super(TinkerGadgets.flintShurikenEntity.get(), throwerIn, worldIn);
+  }
+
+  public FlintShurikenEntity(EntityType<? extends QuartzShurikenEntity> type, LivingEntity livingEntityIn, World worldIn) {
```

Comment:
Looks like the wrong entity type there, should be Flint. Though, are these two constructors taking an entity type even used? Can probably ditch them, they are mostly for inheritance (and extensions should use `ShurikenEntityBase`)

Reply:
Fixed. That constructor wasn't even needed, may bad.